{{cookiecutter.package_name}}/# COOKIECUTTER{% if cookiecutter.docker_image == "y" %}Dockerfile# COOKIECUTTER{% endif %}

```diff
@@ -0,0 +1,34 @@
+ARG PYTHON_VERSION=3.12
+FROM python:${PYTHON_VERSION}-alpine AS builder
```

Comment:
For actively developed project this is true, but for dockerized CLIs alpine is has additional benefits, since its twice as small, which is reflected in bootup speed. There is some argument for security as well (less security alerts due less of stuff included by default).

Reply:
One thing worth noting, is project using this template can always replace this Dockerfile if for some reason alpine proves a non-starter. And then it could even be added as another option to cookiecutter if that becomes a common need.

Right now for two projects were I expect this dockerfile to be used alpine will do better.